### PR TITLE
Add `Roadmap` from Table of Contents.

### PR DESCRIPTION
**Pull request by walreyes**
### What does this PR do?
* Adds `Roadmap` to table of contents.

There is no section for the roadmap. Would love to read one though!

### CODE REVIEW

**Comment by anithri** [comment]
This is a very good point, but I'm not sure the best way to fix things is to remove the section entirely.

Let me take a pass at updating the section.  Do you want to update your PR with this?

### Roadmap

#### 2.0 
  * rename to blueprint-cli
  * replace commander with yargs for cli
  * extend .blueprintrc settings.
  * Allow .blueprintrc files to set search path for blueprint directories
  * Enable npm blueprint packages
  * Enable better options support for blueprint generation
  * Add Copy command to cli
  * Add config command to cli
  * Add blueprintrc and blueprint-package blueprints
  * Update existing blueprints and move into own package, to be included by default

#### 2.X
  * Replace "new" command with alternate ways of project creationg
  * Enable blueprint partials
  * Enable blueprints for assets: icons, images, css, fonts
  * Enable ability to insert text into existing files
  * Enable generators that can invoke other generators.  Inspired by rails scaffold

**Comment by walreyes** [comment]
Done @anithri.

I wanted to read the Roadmap because I am planning to work on a CLI to add redux code, and I wanted to know if `redux-cli` is planning to do something similar.

What I find myself doing a lot it's adding a lot boilerplate code to a `redux` application, like adding a new action, a new value to the reducer, a new constant. So I was thinking that a `cli` could do it for me and ask me what are the values for each thing.

I guess that I could experiment with the `postInstall` hook.

But It'd be great to get variables from the user from command prompts.

File: readme.md [hunk 1]
```diff
@@ -143,15 +143,38 @@ redux g dumb MyNewComponent --dry-run
...
+* Enable npm blueprint packages
+* Enable better options support for blueprint generation
+* Add Copy command to cli
+* Add config command to cli
+* Add blueprintrc and blueprint-package blueprints
+* Update existing blueprints and move into own package, to be included by default
+
+#### 2.X
+
+* Replace "new" command with alternate ways of project creationg
```
Comment (jamesr73) [inline readme.md]: Fix @anithri 's creation_*g*_ typo 😉

**Comment by anithri** [comment]
This is looking good except...  We just renamed .reduxrc to .blueprint rc, and the cli has been renamed to 'bp'.

**Comment by walreyes** [comment]
@anithri I've rebased with master so the readme has a roadmap and the recent name change to `.bluprintrc` and `bc`.

What do you think?

File: readme.md (hunk 1 above)
Comment (walreyes) [reply readme.md]: Thanks @jamesr73 👍. It has been fixed.

**Comment by anithri** [comment]
Thanks!

**Pull request merged**